youtube-channels.csv

```diff
@@ -425,3 +425,13 @@ UCKvCGuur4JFEgScY-81Ph0Q, I AM ME, https://www.youtube.com/channel/UCKvCGuur4JFE
 UCHQs2gBIRQ0oLRHhSV5Jrig, House Nest, https://www.youtube.com/channel/UCHQs2gBIRQ0oLRHhSV5Jrig
 UCCfmUX1OoQVmqalSft-O5BA, Sieben, https://www.youtube.com/channel/UCCfmUX1OoQVmqalSft-O5BA
 UCzxogHW8GLn5W5AYbPmQJOg, Brutalist Empire, https://www.youtube.com/channel/UCzxogHW8GLn5W5AYbPmQJOg
+UCPDILkId0HJTU9A82ItZn2A, X-Dream, https://www.youtube.com/channel/UCPDILkId0HJTU9A82ItZn2A
+UCDo_VHBvW6g6esvbRLyqr8A, DEE BDUBZ, https://www.youtube.com/channel/UCDo_VHBvW6g6esvbRLyqr8A
+UCgZn1e9AK2GMFFL5pqxqPYg, AlexDaftPunk91, https://www.youtube.com/channel/UCgZn1e9AK2GMFFL5pqxqPYg
+UC3REe_sF54K57o_PcC_Pn_A, Aleix Dorian, https://www.youtube.com/channel/UC3REe_sF54K57o_PcC_Pn_A
```

Comment:
This is OK, just a note to myself before merging: need to make sure tracks with format `Dave Tarrida - B2 - Time Squared` are correctly parsed by https://github.com/mirrorfm/trackfilter/

Reply:
I don't think tracks will be recognized for this one. I've added it to https://github.com/mirrorfm/data/issues/37 so we don't forget about it